tests/routes/dive-log.tests.js

```diff
@@ -1,7 +1,7 @@
 import _ from 'lodash';
 import { app } from '../../service/server';
 import Bluebird from 'bluebird';
-import { createUser, purgeTable } from '../test-utils';
```

Comment:
Is the `createUser` util even used anywhere anymore? I think the `generator` made it obsolete.

Reply:
Removing it... it's no longer in use.